src/web3/oracle.ts

```diff
@@ -17,10 +17,9 @@ export async function genDeployAuthProof(
         throw new Error("Not connected");
     }
 
-    const authType = useAuthStore().user!.authType;
     const identityType = useAuthStore().user!.identityType;
     let genAuthProof: HttpsCallable;
-    if (identityType === "twitter.com" && authType === "oauth") {
+    if (identityType === "twitter.com") {
         genAuthProof = httpsCallable(functions, 'genTwitterOAuthProof');
```

Comment:
should we throw here since we removed the genTwitterOAuthProof?

Reply:
Good point. Will update it in another diff along with the update of deployment logic.